app/src/main/java/com/example/tasky/agenda/domain/AddNeedleToAgenda.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.tasky.agenda.domain
```

Comment:
In my opinion, the Needle logic isn't business logic. It's presentation logic

So I would add this on the presentation package